integrations/rename_branch_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"net/http"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRenameBranch(t *testing.T) {
+	// get branch setting page
+	session := loginUser(t, "user2")
+	req := NewRequest(t, "GET", "/user2/repo1/settings/branches")
+	resp := session.MakeRequest(t, req, http.StatusOK)
```

Comment:
That can be refactored once #16052 gets merged. Will then look like
```suggestion
	token := getUserToken(t, "user2")
	req := NewRequest(t, "GET", "/user2/repo1/settings/branches")
	resp := session.MakeRequest(t, WithToken(req, token), http.StatusOK)
```
if I understand the refactoring taken there correctly.